grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -467,6 +566,10 @@ public GrpcService build() {
             handlerRegistry = registryBuilder.build();
         }
 
+        if (exceptionMappings != null) {
+            statusFunction = toGrpcStatusFunction(exceptionMappings);
```

Comment:
Could we introduce a local variable? Otherwise a user cannot do this:

```java
builder.addExceptionMapping(...);
GrpcService a = builder.build();
builder.addExceptionMapping(...);
GrpcService b = builder.build();
```

Reply:
Oops... Thanks for pointing out. Fixed